rmgpy/rmg/model.py

```diff
@@ -210,9 +210,11 @@ def __init__(self, core=None, edge=None):
             self.edge = ReactionModel()
         else:
             self.edge = edge
-        
-        self.surfaceSpecies = []
-        self.surfaceReactions = []
+        if surface is None:
```

Comment:
I am not super knowledgeable about surface, but should we be creating a surface object at all if the user turns the surface off? This might be good to note for a overhaul of model.py

Reply:
Turning the surface "off" right now is unrelated to the presence of the surface object.  Surface calculations are turned "on" and "off" based on whether or not a surface related tolerance was set not based on the presence of a surface object.  If the surface is "off" the surface object will just always be empty.  We also don't pass model objects directly to reactionSystem objects so without a larger overhaul we still have to rely on the tolerances to tell if it should be "off" or "on".  I suppose if you're expressing concern about instantiating an object we don't use I can leave it as none and add if-statements to manage it, but I don't think it would improve speed or readability much.